kore/src/main/scala/org/kframework/definition/outer.scala

```diff
@@ -223,7 +223,8 @@ case class Module(val name: String, val imports: Set[Module], localSentences: Se
   @transient lazy val signatureFor: Map[KLabel, Set[(Seq[Sort], Sort)]] =
     productionsFor mapValues {
       ps: Set[Production] =>
-        ps.map {
+        ps.filter { p: Production => p.params.isEmpty }
+        .map {
```

Comment:
Parametric productions aren't considered part of the signature? Or will the resolved variants of them already also be present in the definition whenever this is called?

Reply:
the latter.